src/test/ui/invalid/invalid-crate-type.stderr

```diff
@@ -1,10 +1,58 @@
 error: invalid `crate_type` value
-  --> $DIR/invalid-crate-type.rs:12:1
+  --> $DIR/invalid-crate-type.rs:12:15
    |
 LL | #![crate_type="foo"]    //~ ERROR invalid `crate_type` value
-   | ^^^^^^^^^^^^^^^^^^^^
+   |               ^^^^^
    |
    = note: #[deny(unknown_crate_types)] on by default
 
-error: aborting due to previous error
+error: invalid `crate_type` value
+  --> $DIR/invalid-crate-type.rs:16:15
+   |
+LL | #![crate_type="statoclib"]
+   |               ^^^^^^^^^^^ help: did you mean: `staticlib`
```

Comment:
Notice that the span includes the `"`s, but the suggestion doesn't, which would make applying this suggestion generate the code `#![crate_type=staticlib]`, which is incorrect. Easiest solution is to include the quotes in the suggestion.